g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/v2/image/ImageAPI.java

```diff
@@ -42,17 +42,17 @@ public class ImageAPI {
     public void getCountryFlagWithRegion(
             @Parameter(name = APIParamName.REGION, required = true, description = APIParamValue.REGION) @RequestParam(value = APIParamName.REGION, required = true) String region,
             @Parameter(name = APIParamName.SCALE, required = false, description = APIParamValue.COUNTRY_FLAG_SCALE) @RequestParam(value = APIParamName.SCALE, required = false) Integer scale,
-            @Parameter(name = APIParamName.TYPE, required = false, description = APIParamValue.TYPE) @RequestParam(value = APIParamName.TYPE, required = false) String type,
+            @Parameter(name = APIParamName.IMAGE_TYPE, required = false, description = APIParamValue.IMAGE_TYPE) @RequestParam(value = APIParamName.IMAGE_TYPE, required = false) String image_type,
```

Comment:
Why change `type` to `image_type`? This is redundant because flags are surely images.

Reply:
Colin suggest that change type to image_type

Reply:
Actually 'json' isn't a valid image type. This design needs to be improved.

Reply:
It mean response format